src/foam/nanos/cron/Schedulable.js

```diff
@@ -60,6 +68,55 @@
       javaCode: `
         // TODO: submitting the entry to the dao
       `
+    },
+    {
+      name: 'authorizeOnCreate',
+      javaCode: `
+        // not authorized for now
```

Comment:
believe basicUser (and sme) should have the ability to create. 
Assume this is incomplete.

Reply:
yeah this is still todo, currently all users are able to create. adding to basicUser make sense, just need to check if the appropriate groups are extending basicUser